information.js

```diff
@@ -1,75 +1,101 @@
-        longitude : 65.01383,
-        latitude : 25.46965,
-        address : "Torikatu 10, 90100 Oulu",
-        businesshours : "Oulu10\nMonday-Friday: 9am-4pm"
-    },
-    bikeStuff : {
-        text : "There are various bike stores spread around in Oulu. A bicycle is often the main means of transportation for a student. Not only is it a healthy form of everyday exercise, but it can also save you money over taking a bus, and time if the option were to walk. In recent years, there was a push for rentable city bikes, too, but it is still figuring itself out.",
-        longitude : 65.02561,
-        latitude : 25.47724,
-        address : "Valtatie 49, 90500 Oulu",
-        businesshours : "Jussin Pyöräpiste\nMonday-Friday: 9am-6pm\nSaturday: 9am-2pm"
-    },
-    yths : {
-        text : "The student healthcare is located right next to the university, providing vital services such as dentistry, general practice, and mental health services. If you are a student, you have the student healthcare fee as well, so you are viable to use their services. For more serious and acute ailments, the actual hospital is probably a better choice, despite being a good distance away from the campus region.",
-        longitude : 65.0578412742499,
-        latitude : 25.471201639275847,
-        address : "Yliopistokatu 1, 90570 Oulu",
-        businesshours : "Monday-Thursday: 8am-3pm\nFriday: 8am-2pm"
-    },
-    teekkaritalo : {
-        text : "Teekkaritalo often serves as a partying place for various student parties that take place outside the city centre and/or night clubs. It is sizable enough to house the active students of larger guilds, but smaller guilds also collaborate to fill it up to its full capacity. Due to its roots as being a place specifically built by the technology students back in the day, it is very student friendly by offering a decent premise at an affordable cost, complete with two saunas and a yard with a fireplace at a gazebo.",
-        longitude : 65.06384497095632,
-        latitude : 25.483870212287922,
-        address : "Kalervontie 7, 90570 Oulu"
-    }
-}
\ No newline at end of file
+const places = [
+  {
+    name: "University of Oulu",
+    text:
+      "The University of Oulu. A location of learning for thousands of students from various walks of life. The campus houses several restaurants and other establishments, including a few libraries and even less official 'student spaces' for extra curricular activities.",
+    longitude: 65.05927696679123,
+    latitude: 25.466218396946577,
+    address: "Pentti Kaiteran katu 1, 90570 Oulu",
+    picture:
+      "https://pbs.twimg.com/profile_images/1207983328458096640/nSMq6zr5_400x400.png",
```

Comment:
Twitter's terms of service allows the platform to use the photo or video but not anyone else